### PR TITLE
Make conversationId optional

### PR DESCRIPTION
**Pull request by abdullahselek**
- This commit fixes the issue #27

### CODE REVIEW

File: iyzipay/iyzipay_resource.py [hunk 1]
```diff
@@ -68,7 +68,10 @@ def format_header_string(api_key, hashed):
 
     @staticmethod
     def resource_pki(request):
-        return 'locale=' + request.get('locale') + ',conversationId=' + request.get('conversationId') + ','
+        if request.get('conversationId') != None:
```
Comment (kadriyedogan) [inline iyzipay/iyzipay_resource.py]: Hi,  @abdullahselek 
You should verify this implementation with unit test which sends conversationId null or empty.

Comment (abdullahselek) [reply iyzipay/iyzipay_resource.py]: Hi @kadriyedogan, I couldn't see any unit test target for this library. Am I missing something?

Comment (kadriyedogan) [reply iyzipay/iyzipay_resource.py]: @abdullahselek Sorry, my mistake. We added unit tests for other clients such as iyzipay-java, iyzipay-ruby, iyzipay-dotnet except python client :) I see that we only have samples.
 I will create another issue to add test support.If you have any experience about unit testing and contribute to us we would be happy 👍

Comment (abdullahselek) [reply iyzipay/iyzipay_resource.py]: @kadriyedogan no worries. When you have time please review my changes. For the new issue adding test support I'm going to make a new PR.

**Comment by abdullahselek** [comment]
@yyenigun thanks for reviewing. It seems I don't have write access could you please merge my PR?

File: iyzipay/iyzipay_resource.py [hunk 2]
```diff
@@ -68,7 +68,10 @@ def format_header_string(api_key, hashed):
...
+            return 'locale=' + request.get('locale') + ',conversationId=' + request.get('conversationId') + ','
+        else:
+            return 'locale=' + request.get('locale') + ','
```
Comment (mertkahyaoglu) [inline iyzipay/iyzipay_resource.py]: how about ?
```python
return 'locale=' + request.get('locale') + (',conversationId=' + request.get('conversationId') + ',' if request.get('conversationId') else ',')
```

Comment (abdullahselek) [reply iyzipay/iyzipay_resource.py]: Yes that's better, thanks. I'm going to update my PR.

**Pull request merged**